wordcloud/views.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from django.http import HttpResponse
+from django.utils import simplejson
+from django.conf import settings
+
+from .wordcloud import popular_words
+
+
+def wordcloud(request, max_entries=30):
+    """ Return tag cloud JSON results"""
+    cache_path = settings.WORDCLOUD_CACHE_PATH
+    if os.path.exists(cache_path):
+        with open(cache_path) as cached_file:
+            content = cached_file.read()
```

Comment:
If there's some long caching set on this (and Ian's using Varnish or some other caching reverse proxy) it may not matter, but this looks like a case for using x-sendfile to serve the generated file more efficiently.